### PR TITLE
#1039 - one-way attribute reflection (attribute -> property)

### PR DESCRIPTION
**Pull request by treshugart**
Fixes #1939.

### CODE REVIEW

File: test/unit/api/properties.js [hunk 1]
```diff
@@ -31,7 +31,7 @@ function testTypeValues (type, values, done) {
   }, 1);
 }
 
-describe('api/prop', () => {
+describe.only('api/prop', () => {
```
Comment (Hotell) [inline test/unit/api/properties.js]: remove `only`

Comment (treshugart) [inline test/unit/api/properties.js]: Sigh. As much as I hate eslint config, semistandard isn't cutting it :p

Comment (treshugart) [inline test/unit/api/properties.js]: done

**Comment by treshugart** [comment]
Updated docs!

**Comment by treshugart** [comment]
Will merge now (since I've fixed stuff and you're probably asleep) but happy to fix any lingering stuff in 5.x after the fact. Cheers!

**Comment by Hotell** [comment]
Nice! Oh it's only 11am in Prague so all good. Haha

**Pull request merged**